src/WebService.cpp

```diff
@@ -0,0 +1,117 @@
+#include "WebService.h"
+
+namespace weatherserver {
+
+    const uint16_t WebService::OPC_NS_INDEX = 1;
+    const utility::string_t WebService::ENDPOINT_API_OPENAQ = U("https://api.openaq.org/v1/");
+    const utility::string_t WebService::PATH_API_OPENAQ_COUNTRIES = U("countries");
+    const utility::string_t WebService::PATH_API_OPENAQ_LOCATIONS = U("locations");
+    const utility::string_t WebService::PATH_API_OPENAQ_MEASUREMENTS = U("measurements");
+    const utility::string_t WebService::PARAM_API_OPENAQ_COUNTRY = U("country");
+    const utility::string_t WebService::PARAM_API_OPENAQ_LIMIT = U("limit");
+
+    const utility::string_t WebService::ENDPOINT_API_DARKSKY = U("https://api.darksky.net/forecast");
+    const utility::string_t WebService::PARAM_API_DARKSKY_EXCLUDE = U("exclude");
+    const utility::string_t WebService::PARAM_API_DARKSKY_UNITS = U("units");
+    const std::string WebService::PARAM_VALUE_API_DARKSKY_MINUTELY = "minutely";
+    const std::string WebService::PARAM_VALUE_API_DARKSKY_HOURLY = "hourly";
+    const std::string WebService::PARAM_VALUE_API_DARKSKY_DAILY = "daily";
+
+    WebService::WebService(Settings* settingsObj) {
+        settings = settingsObj;
```

Comment:
It is probably ok to go with pointer here. But `nullptr` can be passed and later some function of class may crash because of this.

If we make settings copyable (and it is) then we can pass const reference and have a proper copy in this class:
```c++
WebService::WebService(const Settings& settingsObj)
    : settings(settingsObj) {}
```
This is called [member initializer list](https://en.cppreference.com/w/cpp/language/initializer_list).